site/app/controllers/forum/ForumController.php

```diff
@@ -560,6 +564,51 @@ public function publishPost() {
                 $result['next_page'] = $this->core->buildCourseUrl(['forum', 'threads', $thread_id]) . '?' . http_build_query(['option' => $display_option]);
                 $result['post_id'] = $post_id;
                 $result['thread_id'] = $thread_id;
+                $posts = $this->core->getQueries()->getPostsInThreads([$result['thread_id']]);
```

Comment:
The following block of code appears to be almost duplicated around line 843 of this file.  Can these be combined?

Reply:
Fixed in https://github.com/Submitty/Submitty/pull/9906/commits/dc8a27b9f24bcc2c03816900802567d723db7180